datahub-web-react/src/app/entity/dataJob/preview/Preview.tsx

```diff
@@ -68,6 +70,8 @@ export const Preview = ({
             urn={urn}
             description={description || ''}
             type="Data Task"
+            entityType={EntityType.DataJob}
```

Comment:
Might have to merge the refactor I did around here first...

Reply:
please send me the commit

Reply:
thats part of a bigger refactoring you did.

for now, i'm going to keep the current convention until we decide how to proceed here

Reply:
I think we should merge that in; I can work on that myself later